lib/ui/views/groups/group_details_view.dart

```diff
@@ -174,9 +178,11 @@ class _GroupDetailsViewState extends State<GroupDetailsView> {
                   child: Text('CANCEL'),
                   onPressed: () => Navigator.pop(context),
                 ),
-                FlatButton(
-                  child: Text('ADD'),
-                  onPressed: () => onAddGroupMemberPressed(context),
+                CVFlatButton(
+                  triggerFunction: onAddGroupMemberPressed,
+                  context: context,
+                  buttonText: 'ADD',
+                  key: addButtonGlobalKey,
```

Comment:
Key as the first param should be there.

Reply:
@Nitish145 Check for requested changes in latest commits.